docs/translations-needed.md

```diff
@@ -329,8 +329,7 @@ This default value is hardcoded as a default to the localize method in the SeekB
 |                         | {1} is loading.                                                                     |
 |                         | Exit Picture-in-Picture                                                             |
 |                         | Picture-in-Picture                                                                  |
-| es.json (missing 2)     | Exit Picture-in-Picture                                                             |
-|                         | Picture-in-Picture                                                                  |
+| es.json (Complete)      |                                                                                     |
```

Comment:
This was picked up when running: `npm run docs:lang` It must have been missed with a bug fix.